src/script/deploy.js

```diff
@@ -0,0 +1,283 @@
+      return {};
+    }
+    try {
+      const fileContents = JSON.parse(fs.readFileSync(CONFIG_FILENAME).toString());
+      if (fileContents) {
+        spinner.succeed("Configuration loaded");
+        return fileContents;
+      } else {
+        spinner.warn("Unable to parse configuration: deleting and continuing");
+        fs.rmSync(CONFIG_FILENAME);
+        return {};
+      }
+    } catch {
+      spinner.warn("Unable to parse configuration: deleting and continuing");
+      fs.rmSync(CONFIG_FILENAME);
+      return {};
+    }
+  } else {
+    spinner.succeed("Configuration not loaded");
+    return {};
+  }
+}
+
+async function saveConfiguration(config) {
+  const oldData = (() => {
+    try {
+      return JSON.parse(fs.readFileSync(CONFIG_FILENAME).toString());
+    } catch {
+      return {};
+    }
+  })();
+
+  const data = JSON.stringify({ ...oldData, ...config });
+  fs.writeFileSync(CONFIG_FILENAME, data);
+}
+
+///////////////////////////////////////////////////////////////////
+///                         DEPLOYMENTS                         ///
+///////////////////////////////////////////////////////////////////
+
+async function deployLineaWorldID(config) {
+  const spinner = ora("Deploying LineaID on Linea...").start();
+
+  try {
+    const data = execSync(
+      `forge script src/script/DeployLineaWorldID.s.sol:DeployOpWorldID --fork-url ${config.lineaRpcUrl} \
+      --etherscan-api-key ${config.lineaEtherscanApiKey} --broadcast --verify -vvvv`,
+    );
+    console.log(data.toString());
+  } catch (err) {
+    console.error(err);
+  }
+
+  spinner.succeed("DeployLineaWorldID.s.sol ran successfully!");
+}
+
+///////////////////////////////////////////////////////////////////
+///                      MAINNET DEPLOYMENT                     ///
+///////////////////////////////////////////////////////////////////
+
+async function deployLineaStateBridgeMainnet(config) {
+  const spinner = ora("Deploying Linea State Bridge...").start();
+
+  try {
+    const data =
+      execSync(`forge script src/script/DeployLineaStateBridgeMainnet.s.sol:DeployLineaStateBridgeMainnet --fork-url ${config.ethereumRpcUrl} \
+      --etherscan-api-key ${config.ethereumEtherscanApiKey} --broadcast --verify -vvvv`);
+    console.log(data.toString());
+  } catch (err) {
+    console.error(err);
+  }
+
+  spinner.succeed("DeployLineaStateBridgeMainnet.s.sol ran successfully!");
+}
+
+///////////////////////////////////////////////////////////////////
+///                          INITIALIZE                         ///
+///////////////////////////////////////////////////////////////////
+
+async function InitializeLineaStateBridge(config) {
+  const spinner = ora("Initializing LineaStateBridge...").start();
+
+  try {
+    const data = execSync(
+      `forge script src/script/InitializeLineaStateBridge.s.sol:LineaStateBridge --fork-url ${config.lineaRpcUrl} --broadcast -vvvv --legacy`,
```

Comment:
Same here, this script will need to know addresses of both `LineaWorldID` and `LineaStateBridge` to run.